core/templates/dev/head/pages/exploration_editor/translation_tab/TranslatorOverviewDirective.js

```diff
@@ -79,6 +79,19 @@ oppia.directive('translatorOverview', [
             $window.localStorage.setItem(
               LAST_SELECTED_TRANSLATION_LANGUAGE, $scope.languageCode);
           };
+
+          $scope.label = function() {
+            if ($scope.numberOfRequiredAudio -
+              $scope.numberOfAudioNotAvailable < 1) {
+              return $scope.numberOfRequiredAudio -
+                $scope.numberOfAudioNotAvailable + ' item translated out of ' +
+                $scope.numberOfRequiredAudio + ' items';
+            } else {
+              return $scope.numberOfRequiredAudio -
+                $scope.numberOfAudioNotAvailable + ' items translated out of' +
+                ' ' + $scope.numberOfRequiredAudio + ' items';
```

Comment:
Move this (`' '`) space to the above line?

```
                $scope.numberOfAudioNotAvailable + ' items translated out of ' +
                $scope.numberOfRequiredAudio + ' items';
```

Reply:
It will exceed the max length of a line.

Reply:
![image](https://user-images.githubusercontent.com/16653571/54958833-2a423880-4f7d-11e9-9423-13907a68549f.png)

@suicide11, It's working fine for me, have I missed something?

Reply:
In my editor it is showing line 91 column 81

Reply:
Ok I will do it in another way.